variables.tf

```diff
@@ -70,22 +75,20 @@ variable "private_subnet_tags" {
   }
 }
 
+# -------------------------------------------------------------------------------------------------
+# Bastion Host (optional)
+# -------------------------------------------------------------------------------------------------
 variable "bastion_name" {
   description = "If not empty will overwrite the bastion host name specified by 'name'"
   default     = ""
 }
 
-# -------------------------------------------------------------------------------------------------
-# Bastion Host (required)
-# -------------------------------------------------------------------------------------------------
 variable "bastion_ssh_keys" {
   description = "A list of public ssh keys to add to authorized_keys file"
   type        = "list"
+  default     = ["0.0.0.0/0"]
```

Comment:
Should be an empty list.

Reply:
Good catch. :+1: for security awareness

Reply:
@maartenvanderhoef also note that we try to be as picky and accurate as we can, especially on standalone stuff (ansible roles, TF modules). So don't hold yourself back here.